tests/integration/auth.spec.coffee

```diff
@@ -42,6 +42,13 @@ describe 'SDK authentication', ->
 
 				m.chai.expect(promise).to.be.rejectedWith('Unauthorized')
 
+		describe 'resin.auth.getToken()', ->
+
+			it 'shoud be rejected', ->
```

Comment:
Typo: `shuod` -> `should`

Reply:
🚓 Typo there Tim

Reply:
I'm going to squash all these commits to one. Next time will do before creating the PR. Thank you :)